File: aspnet-core/src/KiemKeDatDai.Core/EntitiesDb/DonViHanhChinh.cs
using Abp.Domain.Entities.Auditing;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace KiemKeDatDai.EntitiesDb
{
    [Table("DonViHanhChinh")]
    public class DonViHanhChinh : FullAuditedEntity<long>
    {
        public string TenVung { get; set; }
        public string MaVung { get; set; }
        public string TenTinh { get; set; }
        public string MaTinh { get; set; }
        public string TenHuyen { get; set; }
        public string MaHuyen { get; set; }
        public string TenXa { get; set; }
        public string MaXa { get; set; }
        public string Name { get; set; }
        public long? Parent_id { get; set; }
        public string Parent_Code { get; set; }
        public long CapDVHCId { get; set; }
        public bool Active { get; set; }
        public long Year { get; set; }
        public int? TrangThaiDuyet { get; set; }
        public DateTime? NgayGui { get; set; }
        public DateTime? NgayDuyet { get; set; }
        public int? SoDVHCCon { get; set; }
        public int? SoDVHCDaDuyet { get; set; }
    }
}

File: src/services/dvhc/dto/DonViHanhChinh.ts
export interface DonViHanhChinh {
    childStatus?: number;
    tenTinh?: string;
    maTinh?: string;
    tenHuyen?: string | null;
    maHuyen?: string | null;
    tenXa?: string | null;
    maXa?: string | null;
    maVung?: string | null;
    name: string;
    parent_id?: number;
    parent_Code?: string | null;
    capDVHCId?: number;
    active?: boolean;
    year_Id?: number;
    trangThaiDuyet?: number;
    isDeleted?: boolean;
    deleterUserId?: number | null;
    deletionTime?: string | null;
    lastModificationTime?: string | null;
    lastModifierUserId?: number | null;
    creationTime?: string;
    creatorUserId?: number | null;
    id?: number;
    isExitsUser?:any
}